drivers/nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -847,19 +864,6 @@ enum op_band {
 	BAND_24G
 };
 
-/**
- * @brief This enum specifies the type of frames used to retrieve buffered data
- *  from the AP in power save mode.
- */
-enum data_retrieve_mechanism {
-	/** Retrieves data from the AP using a PS-Poll frame */
-	PS_POLL_FRAME,
-	/** Retrieves data from the AP using a QoS Null frame */
-	QOS_NULL_FRAME,
-	/** For future implementation. The RPU will decide which frame to use */
-	AUTOMATIC
-};
-
```

Comment:
Why are these re removed? If it's intended please do this in a separate commit with proper commit log.

Reply:
Handled in separate commit